### ISSUE TITLE
Import `multiclassPairs` as a dependency

### ISSUE THREAD

**Issue by mattssca** [LundBladderCancerGroup/LundTaxonomy2023Classifier#10]
Maybe it's worth exploring adding `multiclassPairs` as a dependency for this package. Looks like a few of the functions in this package are direct duplicates of already available functions in the `multiclassPairs` package. Namely `multiclassPairs::ReadData` and `multiclassPairs::predict_RF`. If we decide this is a good approach, I can add this dependency and remove the duplicated code.

**Comment by elenaara** [comment]
I agree, at first I added the duplicated functions to avoid importing the package but I think it is better to just have multiclassPairs. Feel free to make the change, I was also about to remove those functions as I was doing the other changes.

**Comment by mattssca** [comment]
Thanks, I can take care of this.

**Comment by mattssca** [comment]
This is fixed in #15 . Closing this issue once that PR is merged.
